apps/src/util/dismissVersionRedirect.js

```diff
@@ -0,0 +1,38 @@
+const dismissVersionRedirect = module.exports;
```

Comment:
This looks like the old way of exporting things. Instead, you could `export const DISMISSED_REDIRECT_WARNINGS_SESSION_KEY ...`, and then in the other file, `import * as dismissVersionRedirect from ...`. unless there's a reason that's not working.

Reply:
looks like the constant I used as an example isn't exported. so a better example might be 
`export const dismissedRedirect = function(...) {` or `export function dismissedRedirect(...) {`

Reply:
> `then in the other file, import * as dismissVersionRedirect from ...`

Actually, it seems like it would be even better to import the items individually, as 
```js
import {dismissedRedirectWarning, dismissedRedirectDialog, ...} from ...`
```
sorry to go around so many times on this 😳